test/e2e/types/testnet.go

```diff
@@ -80,6 +90,15 @@ func (o OmniEVM) BootNodesStr() string {
 	return strings.Join(resp, ",")
 }
 
+func (o OmniEVM) BootNodesStrArr() string {
```

Comment:
add a comment explain the different between this and above please, also add comment above please